src/components/DecisioningEngine/index.js

```diff
@@ -15,17 +15,21 @@ import createDecisionProvider from "./createDecisionProvider";
 import createApplyResponse from "./createApplyResponse";
 import createEventRegistry from "./createEventRegistry";
 import createContextProvider from "./createContextProvider";
+import injectWindow from "./injectWindow";
 
 const createDecisioningEngine = ({ config, createNamespacedStorage }) => {
   const { orgId } = config;
   const storage = createNamespacedStorage(
     `${sanitizeOrgIdForCookieName(orgId)}.decisioning.`
   );
-
+  const currentWindow = injectWindow(window);
```

Comment:
i don't see the benefit in creating a new window object, rather than passing in window directly.

Reply:
Yes, 👍